modules.d/95nvmf/nvmf-autoconnect.sh

```diff
@@ -1,5 +1,54 @@
-#!/bin/bash
+#!/bin/sh
+# Argument $1 is "settled", "online", or "timeout", indicating
+# the queue from which the script is called.
+# In the "timeout" case, try everything.
+# Otherwise, try options according to the priorities below.
 
-[ -f /sys/class/fc/fc_udev_device/nvme_discovery ] || exit 1
-echo add > /sys/class/fc/fc_udev_device/nvme_discovery
+[ "$RD_DEBUG" != yes ] || set -x
+
+if [ "$1" = timeout ]; then
+    [ ! -f /sys/class/fc/fc_udev_device/nvme_discovery ] \
```

Comment:
Just a note: same usage of `-f` vs `-e` as stated above.

Reply:
Again, `-f` is correct here. If this existed and was anything else but a regular file, we'd have to ignore it.

Reply:
Ack, thanks!